Building block view

laura-pb opened the pull request:
Point 5 of the documentation done (issue #15)

UO277653 reviewed (changes requested):
I think that it is very well done, taking into account the point at which we are in the development process. The only changes I would suggest would be to add an arrow between User and the Pod in the diagram buildingblock_level2.png, and some small typos that I have found such as these (the suggestions are between brackets):

`| _Data Base_ | Data base used by the DeDe system to store the needed information, such as the orders done, the adress (address), etc.
| _User_| Person accesing (accessing) to the DeDe system and that has its own Pod.`

Apart from that, the rest is fine 🙂

laura-pb commented:
@UO277653 My idea in the level 2 diagram was to just showcase the interactions of the DeDe system (the whitebox), but since that arrow is the only one missing, it's true that it's not clear enough. I added it and fixed the typos 🙂

UO277653 reviewed (approved):
I think that it is fine!
